tests/integration/acp/dac/add_policy/with_invalid_relations_test.go

```diff
@@ -24,24 +24,16 @@ func TestACP_AddPolicy_NoRelations_Error(t *testing.T) {
 				Identity: testUtils.ClientIdentity(1),
 
 				Policy: `
-                    name: a policy
-                    description: a policy
-
-                    actor:
-                      name: actor
-
-                    resources:
-                      users:
-                        permissions:
-                          update:
-                            expr: owner
-                          delete:
-                            expr: owner
-                          read:
-                            expr: owner + reader
-
-                        relations:
-                `,
+description: a policy
+name: a policy
+resources:
+- name: users
+  permissions:
+  - name: delete
+  - expr: reader
+    name: read
+  - name: update
+`,
```

Comment:
todo: The automated script made this and below (TestACP_AddPolicy_NoRelationsLabel_Error) test duplicates / clones.

Here it should be testing this:
```suggestion
description: a policy
name: a policy
resources:
- name: users
  permissions:
  - name: delete
  - expr: reader
    name: read
  - name: update
  relations:
`,
```